static/js/common/upload-addon.js

```diff
@@ -191,6 +191,14 @@
                 upload_progress_outside.attr('class', 'bar-fail');
                 upload_progress_inside.fadeOut();
 
+                if (results && results.processed_by_addons_linter) {
+                    $("<a>").text(gettext('We have enabled a new linter to process your Add-on. Please make sure to report any issues on GitHub'))
+                            .attr('href', 'https://github.com/mozilla/addons-linter/')
+                            .attr('class', 'addons-linter-info')
+                            .attr('target', '_blank')
```

Comment:
indeed, there are other places too that use it. Maybe i'll try to fix them all (or most of them) in a separate PR